src/main/java/forestry/core/DefaultClimateProvider.java

```diff
@@ -0,0 +1,34 @@
+package forestry.core;
+
+import forestry.api.core.EnumHumidity;
+import forestry.api.core.EnumTemperature;
+import forestry.api.core.ForestryAPI;
+import forestry.api.core.IClimateProvider;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+import net.minecraft.world.biome.Biome;
+
+public class DefaultClimateProvider implements IClimateProvider {
+    private World world;
+    private BlockPos pos;
+
+    public DefaultClimateProvider(World world, BlockPos pos) {
+        this.world = world;
+        this.pos = pos;
+    }
+
+    @Override
+    public Biome getBiome() {
+        return world.getBiome(pos);
+    }
+
+    @Override
+    public EnumTemperature getTemperature() {
+        return EnumTemperature.getFromValue(ForestryAPI.climateManager.getTemperature(world, pos));
```

Comment:
Replace this line with `EnumTemperature.getFromBiome(Biome Biome, World world, BlockPos pos)` here or it can't detect `EnumTemperature.HELLISH`